tox.ini

```
@@ -22,19 +29,19 @@ commands =
     mypy src/ansible_builder
     pylint src/ansible_builder test
 
-[testenv:unit{,-py39,-py310,-py311,-py312}]
+[testenv:unit{,-py39,-py310,-py311,-py312,-py313}]
 description = Run unit tests
 commands = pytest -n auto test/unit {posargs} {[shared]pytest_cov_args}
 
-[testenv:pulp-integration{-py39,-py310,-py311,-py312}]
+[testenv:pulp-integration{-py39,-py310,-py311,-py312,-py313}]
 # Some of these tests must run serially because of a shared resource
 # (the system policy.json file).
 description = Run pulp integration tests
 commands =
     pytest -n auto -m "not serial" test/pulp_integration {posargs} {[shared]pytest_cov_args}
     pytest -n 0 -m "serial" test/pulp_integration {posargs} {[shared]pytest_cov_args}
 
-[testenv:integration{,-py39,-py310,-py311,-py312}]
+[testenv:integration{,-py39,-py310,-py311,-py312,-py313}]
 description = Run integration tests
 # rootless podman reads $HOME
 passenv =
```

This setting overrides the main test env. You may want to inherit from it to let the other var keep being bypassed here:
```suggestion
passenv =
    {[testenv]passenv}
```

---

Hrm, actually, turns out I don't need the `passenv = UNSAFE_PYO3_SKIP_VERSION_CHECK` at all since it is set within the tox environment and will always be passed from that point. I'll just remove that line.